os/include/tinyara/input/touchscreen.h

```diff
@@ -121,7 +124,7 @@
 #define TOUCH_POS_VALID      (1 << 4) /* Hardware provided a valid X/Y position */
 #define TOUCH_PRESSURE_VALID (1 << 5) /* Hardware provided a valid pressure */
 #define TOUCH_SIZE_VALID     (1 << 6) /* Hardware provided a valid H/W contact size */
-
+#define TOUCH_MAX_POINTS 	15
```

Comment:
Let's add an empty line to distinguish context from above.
And please add some description for this. What does this mean?

Reply:
could you check my comment above?

Reply:
I have added a comment for this.